### PR TITLE
Refactor: Move cities controllers to a dedicated folder

### PR DESCRIPTION
**Pull request by FelipeCasalG**
Refactor: Move cities controllers to a dedicated folder

## Description ⭐

This PR moves all the controllers regarding the cities resource to a new folder inside the controllers. The CreateCityController was renamed to StoreCityController in order to follow Laravel's naming conventions.

### Type of change 🏃

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ❓

No testing added to the project yet.

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+use App\Http\Controllers\Cities\DestroyCityController;
+use App\Http\Controllers\Cities\GetAllCitiesController;
+use App\Http\Controllers\Cities\UpdateCityController;
 
 Route::prefix('cities')->name('cities.')->group(function () {
     Route::get('/', GetAllCitiesController::class)->name('index');
-    Route::post('/', CreateCityController::class)->name('store');
-    Route::put('/{id}', UpdateCityController::class)->name('update');
+    Route::post('/', StoreCityController::class)->name('store');
+    Route::patch('/{id}', UpdateCityController::class)->name('update');
```
Comment (gianfranco-rocco) [inline routes/api.php]:
```suggestion
    Route::put('/{id}', UpdateCityController::class)->name('update');
```
`put` was right. The difference between `put` and `patch` is that for `put`, you send all the resource's fields in the request, while with `patch`, you only send specific ones. Here at Light-it, we define update endpoints as `put`.

File: routes/web.php [hunk 2]
```diff
@@ -1,8 +1,7 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
-use App\Http\Controllers\CityController;
-use App\Http\Controllers\ViewCitiesController;
+use App\Http\Controllers\Cities\ViewCitiesController;
```
Comment (gianfranco-rocco) [inline routes/web.php:4]:
```suggestion
```
This controller isn't being used, you may as well remove the import.

Comment (FelipeCasalG) [reply routes/web.php:4]: I don't understand. Am I not using the controller in line 8?

```
Route::get('/cities', ViewCitiesController::class);
```

Comment (gianfranco-rocco) [reply routes/web.php:4]: Oh, sorry, I completely missed that. My bad!

**Pull request merged**